src/components/menu/GitBranchActionMenu.vue

```diff
@@ -13,8 +13,17 @@
         :data-cy="`git-menu-branch-checkout-loader-${branchName}`"
         color="primary"
         indeterminate
-        v-if="loading.checkout"
+        v-if="!current && loading.checkout"
```

Comment:
`current` is a boolean so maybe change it to `isCurrent` -> also specify current what (branch I guess?)

Reply:
done